eclass/gkrellm-plugin.eclass

```diff
@@ -97,5 +107,6 @@ gkrellm-plugin_src_install() {
 	fi
 }
 
-_GKRELLM_PLUGIN_R1=1
+EXPORT_FUNCTIONS src_install
```

Comment:
at the very end, that is, it needs to be **outside** the `fi` inherit guard

Reply:
alright, fixed.

Reply:
(have an idea why we require this?)

Reply:
I'm not entirely sure, but I would guess that we want to ensure that gkrellm-plugin_src_install() is defined only once, but EXPORT_FUNCTIONS should be executed for each `inherit`.

But I would like to know!

Reply:
https://bugs.gentoo.org/399039

Reply:
Thanks!

Reply:
The devmanual doesn't mention this in any way (but also doesn't give examples of an eclass that inherits another eclass). Could be worthwhile addition maybe?

Reply:
Was reading https://devmanual.gentoo.org/eclass-writing/#export-functions